### PR TITLE
Randomly choose bugs for duplicate classifier

### PR DESCRIPTION
**Pull request by ayush-1506**
Fixes : #541

### CODE REVIEW

**Comment by marco-c** [comment]
What are the results of the model after this?

Can you set a seed with random.seed so that we have consistent results when we rerun the training?

**Comment by ayush-1506** [comment]
Results:
```
Accuracy: f0.7074914453495816 (+/- 0.024142515477883368)
Precision: f0.7105973889023783 (+/- 0.03283942278002216)
Recall: f0.6995125348189415 (+/- 0.04555701682845712)
```

**Comment by marco-c** [comment]
OK, now this makes way more sense ;)

**Comment by marco-c** [comment]
Could you post the full results, with the confidence thresholds too?

File: bugbug/models/duplicate.py [hunk 1]
```diff
@@ -13,6 +15,8 @@
 from bugbug import bug_features, bugzilla, feature_cleanup
 from bugbug.model import BugCoupleModel
 
+random.seed(4)
```
Comment (marco-c) [inline bugbug/models/duplicate.py]: Set the seed in get_labels

Comment (marco-c) [reply bugbug/models/duplicate.py]: So that, when users load the model, the seed is not set for them automatically.

**Comment by ayush-1506** [comment]
Entire result: 
```
Cross Validation scores:
Accuracy: f0.7074914453495816 (+/- 0.024142515477883368)
Precision: f0.7105973889023783 (+/- 0.03283942278002216)
Recall: f0.6995125348189415 (+/- 0.04555701682845712)
X_train: (3606, 40236), y_train: (3606,)
X_test: (400, 40236), y_test: (400,)
/home/ayush99/GitHub/bbug/lib/python3.7/site-packages/sklearn/model_selection/_split.py:1978: FutureWarning: The default value of cv will change from 3 to 5 in version 0.22. Specify it explicitly to silence this warning.
  warnings.warn(CV_WARNING, FutureWarning)
No confidence threshold - 400 classified
[[142  61]
 [ 60 137]]
                   pre       rec       spe        f1       geo       iba       sup

          1       0.70      0.70      0.70      0.70      0.70      0.49       203
          0       0.69      0.70      0.70      0.69      0.70      0.49       197

avg / total       0.70      0.70      0.70      0.70      0.70      0.49       400


Confidence threshold > 0.6 - 267 classified
[[109  31]
 [ 29  98]]
                   pre       rec       spe        f1       geo       iba       sup

          1       0.79      0.78      0.77      0.78      0.78      0.60       140
          0       0.76      0.77      0.78      0.77      0.78      0.60       127

avg / total       0.78      0.78      0.77      0.78      0.78      0.60       267


Confidence threshold > 0.7 - 166 classified
[[80 14]
 [12 60]]
                   pre       rec       spe        f1       geo       iba       sup

          1       0.87      0.85      0.83      0.86      0.84      0.71        94
          0       0.81      0.83      0.85      0.82      0.84      0.71        72

avg / total       0.84      0.84      0.84      0.84      0.84      0.71       166


Confidence threshold > 0.8 - 65 classified
[[39  6]
 [ 2 18]]
                   pre       rec       spe        f1       geo       iba       sup

          1       0.95      0.87      0.90      0.91      0.88      0.78        45
          0       0.75      0.90      0.87      0.82      0.88      0.78        20

avg / total       0.89      0.88      0.89      0.88      0.88      0.78        65


Confidence threshold > 0.9 - 9 classified
[[5 0]
 [1 3]]
                   pre       rec       spe        f1       geo       iba       sup

          1       0.83      1.00      0.75      0.91      0.87      0.77         5
          0       1.00      0.75      1.00      0.86      0.87      0.73         4

avg / total       0.91      0.89      0.86      0.89      0.87      0.75         9
```

**Pull request merged**